content/download/upgradeRecipe/upgradeRecipe7.adoc

```diff
@@ -14,7 +14,7 @@ it allows us to make backwards incompatible changes to the public API _for the l
 We kept these backwards incompatible changes to a strict minimum
 (by favoring deprecation over removal) and will not introduce any additional ones during the 7.x era.
 
-Any backwards incompatible changes are annotated with a [.label.label-danger.label-as-badge.label-public-api]#Public API# badge.
+Any backwards incompatible changes are annotated with a [.badge.label-public-api]#Public API# badge.
```

Comment:
For semantic reasons I guess, as major/minor entries have their own CSS class. I 'll investigate.

Reply:
It's a separate color (purple) from the out-of-the-box ones in bootstrap. But it should be named bg-public-api instead of label-public-api, to align with bg-danger etc. Fixing that.